DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.WebAPI/EventParsers/BaseEventParser.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Newtonsoft.Json.Linq;
+
+namespace GitHubExtension.Activity.External.WebAPI.EventParsers
+{
+    public abstract class BaseEventParser
+    {
+        public abstract JToken ParseEvent(JToken eventToken);
+
+        protected JToken InitialiseEventData(JToken eventToken)
```

Comment:
I do like strongly typed methods, but github has 25 types of events and each of them has different fileld payload, so I don't know is it ok to write around 30+ classes for models